web/models/user.ex

```diff
@@ -78,11 +81,22 @@ defmodule CodeCorps.User do
   def update_changeset(struct, params) do
     struct
     |> changeset(params)
-    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data])
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data, :state_transition])
```

Comment:
Wait, I missed that the error is added to `state_transition`, so this _is_ redundant. Sorry about that.